sharepoint/sharepoint-ps/sharepoint-online/Connect-SPOService.md

```diff
@@ -85,12 +85,20 @@ Connect-SPOService -Url https://contoso-admin.sharepoint.com -Region ITAR
 
 Connects to a SharePoint Online Administration Center specifying the region.
 
+### -----------------------EXAMPLE 5-----------------------------
+
+ ```powershell
+Connect-SPOService -Credential $creds -Url https://tenant-admin.sharepoint.com -ModernAuth $true -AuthenticationUrl https://login.microsoftonline.com/organizations
```

Comment:
Please enclose the PS code with a triple-backtick:
https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide?view=powershell-7.2#markdown-for-code-samples